tests/unittest/_torch/executor/test_kv_cache_budget_split.py

```
@@ -148,7 +151,7 @@ def test_zero_host_cache_unchanged(self):
             target_kv_per_token=80,
         )
 
-        draft_config = c._split_kv_cache_budget_for_draft()
+        target_config, draft_config = c._split_kv_cache_budget_for_draft()
```

Ruff (0.15.15)

[warning] Unpacked variable `target_config` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)